src/components/review/ProposalQuestionaryReview.tsx

```diff
@@ -21,8 +21,16 @@ export default function ProposalQuestionaryReview(
 
   const users = data.users || [];
 
+  const hasReferenceNumberFormat = data.call?.referenceNumberFormat;
```

Comment:
Also here maybe it's better to use `!!data.call?.referenceNumberFormat` to get a boolean value.